### PR TITLE
fix(acbs-durable-functions): resolve durable functions container startup error

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction :pencil2:
Durable functions `acbs` Azure Functions container was failing due to lack of `TS` transpile to `JS`.

## Resolution :heavy_check_mark:
* Webpack installation could scope creep the resolution will be time intensive, as a swift resolution constant have been introduced to `acbs-functions` as JS.
* Removed `@ukef/dtfs2-common` package dependency.

## Miscellaneous :heavy_plus_sign:
* Dockerfile: NPM cache clean has been added post installation.
* All `"@ukef/dtfs2-common` package version has been explicitly set to specific version `1.0.0`.

### CODE REVIEW

File: azure-functions/acbs-function/Dockerfile [hunk 1]
```diff
@@ -14,5 +14,6 @@ WORKDIR /home/site/wwwroot
 COPY package.json .
 
 RUN npm i
+RUN npm cache clean --force
```
Comment (MarRobSoftwire) [inline azure-functions/acbs-function/Dockerfile:17]: Out of interest, why is this needed?

**Review by MarRobSoftwire — DISMISSED** [review]

File: azure-functions/acbs-function/Dockerfile (hunk 1 above)
Comment (abhi-markan) [reply azure-functions/acbs-function/Dockerfile:17]: This is to reclaim the disk space, and requires `--force` flag.
Reference: https://docs.npmjs.com/cli/v7/commands/npm-cache

**Review by avaitonis — APPROVED** [review]
Approved
